### DISCUSSION TITLE
输入框不要那个假的气泡？

### DISCUSSION THREAD

**Discussion by addlistener** [ChatGPTNextWeb/NextChat#633] [Ideas]
<img width="859" alt="image" src="https://user-images.githubusercontent.com/4531670/230724226-b2afc817-94ea-4177-8e0a-c97c32313f02.png">

1. 很占空间，input也不能resize，长的字 input 里面是看不全的
2. 总是有错觉，觉得自己已经按了回车了，过了两分钟才发现没有按
3. 另外有个问题，鼠标移出输入框，focus就消失了。要打字还得再重新点击focus一次

**Comment by ClarenceDan** [comment]
有没有可能就是说，气泡功能在设置里面就能改？focus问题没遇到过

**Reply by addlistener** [reply]
谢谢！看到了

'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Safari/537.36'

输入框是这样，鼠标移出了，就 自动 blur 了，很奇怪

https://user-images.githubusercontent.com/4531670/230841191-109b4dc0-5a4d-44f1-95ff-99e337b0a79c.mov

**Reply by Kosette** [reply]
输入框焦点问题，我记得后面的版本已经改回来了，只在之前的某个版本中会跟随鼠标。

**Reply by addlistener** [reply]
@Kosette 感谢！我按照教程重新fork了一个，发现这个确实已经修复了
